phpBB/phpbb/db/migration/data/v310/update_icons_smilies_url.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v310;
+
+class update_icons_smilies_url extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array(
+			'\phpbb\db\migration\data\v310\rc5',
```

Comment:
If this patch goes into 3.2 only, this needs to be changed and moved.

@Nicofuma in 3.2 or not?